src/Features/Core/Portable/ExtractMethod/SelectionResult.cs

```diff
@@ -138,6 +139,29 @@ public bool ShouldPutAsyncModifier()
             return false;
         }
 
+        public bool ShouldCallConfigureAwaitFalse()
+        {
+            var firstToken = GetFirstTokenInSelection();
+            var lastToken = GetLastTokenInSelection();
+
+            var span = TextSpan.FromBounds(firstToken.SpanStart, lastToken.Span.End);
+
+            foreach (var node in SemanticDocument.Root.DescendantNodesAndSelf())
+            {
+                if (!node.Span.OverlapsWith(span))
+                    continue;
+
+                else if (IsConfigureAwaitFalse(node) && !UnderAnonymousOrLocalMethod(node.GetFirstToken(), firstToken, lastToken))
+                {
```

Comment:
don't mix/match braces in the same cosntruct.  If the `else if` has braces, the `if` should have braces.

Reply:
ooh. Thats horrible. I definitely didn't write that on purpose!